src/components/Display.jsx

```diff
@@ -0,0 +1,15 @@
+import Reviews from "./Reviews";
+import { Routes, Route } from "react-router-dom";
+
+
+const Display = () => {
```

Comment:
Could we consider a better name for this component? Display is very generic, try and be as verbose as possible.